docs/outputs.rst

```diff
@@ -142,7 +142,43 @@ sampled to those subject spaces.
 Confounds
 ---------
 
-See implementation on :mod:`~fmriprep.workflows.bold.confounds.init_bold_confs_wf`.
+The :abbr:`BOLD (blood-oxygen level dependent)` signal measured with fMRI is a mixture of fluctuations
+of both neuronal and non-neuronal origin. Neuronal signals are measured indirectly as changes
```

Comment:
```suggestion
of both neuronal and non-neuronal origin.
Neuronal signals are measured indirectly as changes
```

Please start new sentences in a new line - that helps a lot when looking at future documentation change diffs.

Please modify further instances accordingly, thanks!

Reply:
Thanks! Done.